### ISSUE TITLE
Not all history commands searchable

### ISSUE THREAD

**Issue by rpglover64** [dvorka/hstr#166]
Using `zsh`, 

``` shell
% echo $HISTFILE
/home/alex/.zsh_history
% echo $HH_CONFIG
hicolor
% grep mpv $HISTFILE | wc -l
165
% wc -l <$HISTFILE
20255
```

However, running `hh` and typing "mpv" gives no entries, even cycling through all the match modes with <kbd>C-e</kbd>.

**Comment by razagill** [comment]
@dvorka Any update on this?

**Comment by dvorka** [comment]
Fixed by c36754179965d09b83d72f0bd575a08919208bf5, fix will be released w/ HSTR 2.1
